src/main/java/fi/metatavu/metaform/server/rest/MetaformsApiImpl.java

```diff
@@ -232,7 +246,8 @@ public Response createReply(UUID metaformId, Reply payload, Boolean updateExisti
     }
     
     handleReplyPostPersist(true, metaform, reply, replyEntity, permissionGroups);
-
+    auditLogEntryController.createAuditLogEntry(getLoggerUserId(), AuditLogEntryType.CREATE_REPLY, reply.getId(),
+            null, reply.getId()+" was created by user "+getLoggerUserId());
```

Comment:
use String.format instead of concatenating the string

Reply:
move new audit log entry creation to private method, otherwise we get a lot of duplicated code

Reply:
Move loggedUserId to local variable (getLoggerUserId -method is called repeatedly)